client/client.go

```diff
@@ -200,6 +200,13 @@ type Client struct {
 	Metadata sqlxx.JSONRawMessage `json:"metadata,omitempty" db:"metadata"`
 }
 
+type ClientFilters struct {
```

Comment:
This needs to be added to the swagger spec: 

https://github.com/ory/hydra/blob/b597c88e5ad7e6c00be352c8c641850e5e25f3dc/client/doc.go#L90

You can also probably re-use / re-purpose that struct as `ClientFilter`

Reply:
I added the name/owner params to that struct, is that all that needs done?

Reply:
Yes, that's all that's needed - I think this can be removed then!

Reply:
The ClientFilters struct is used internally to get passed to the client manager, so new filers could be easily added...